contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -633,7 +633,8 @@ def delete_from_changes(self, changes):
 
 
 def dict_if_none(obj, field_name=None):
-    return obj[field_name] if obj[field_name] else {}
```

Comment:
When does a field name not exist here? Doesn't seem like something we should be silently ignoring.

Reply:
When me, prathamesh and Blaine were testing search about 2 months back, there was a field name that wasn't existing so I modified its implementation back then and since on this change has been carried over.

Reply:
Fixed.